### PR TITLE
removing tech leads

### PR DESCRIPTION
**Pull request by xmulligan**
it seems we are moving away from having tech leads to this should be reflected in the governance see #131

### CODE REVIEW

**Comment by iawells** [comment]
I think a better way of describing this is we'll add them when we have something specific for them to do, but I agree that until then we'd be better off not speculating about the process and role.

**Review by iawells — APPROVED** [review]

**Review by taylor — APPROVED** [review]
lgtm

**Review by Smitholi67 — APPROVED** [review]
Lgtm.  Agree that Tech Leads can be added when there becomes a need.

**Comment by taylor** [comment]
@xmulligan fixed merge conflict. check to make sure it looks correct (mainly deleted the sections headers you had deleted that were in main)

**Review by jeffsaelens — APPROVED** [review]
Update looks solid.

**Review by rannyh — CHANGES_REQUESTED** [review]
LGTM

**Comment by sishbi** [comment]
LGTM

**Review by sishbi — APPROVED** [review]

File: GOVERNANCE.md [hunk 1]
```diff
@@ -55,9 +42,7 @@ See [CONTRIBUTING](CONTRIBUTING.md) documentation for more information how to ge
 
 ## Elections
 
-### Timeline
-
-Co-chairs and tech leads will be elected for one year terms and may run for reelection.
+Co-chairs will be elected for one year terms and may run for reelection.
```
Comment (xmulligan) [inline GOVERNANCE.md:47]:
```suggestion
### Timeline

Co-chairs will be elected for one year terms and may run for reelection.
```

File: GOVERNANCE.md [hunk 2]
```diff
@@ -69,11 +56,9 @@ In other words, if two contributors are employed by Company X, two by Company Y,
...
 
-Each organization listed in the [interested parties document](interested-parties.md) has one vote. Interested parties can be added at any time, but must be added at least one week before any election to have a vote. Any contributor from an organization may cast the vote for that organization. Each organization can cast one vote for a co-chair candidate in each of the communities (Kubernetes (K8s) Community, Service Provider (SP) Community, and CNF Developer (Dev) Community), three total votes. Each organization can vote for as many tech leads as they see fit.
-
-### Voting Procedure
+Each organization listed in the [interested parties document](interested-parties.md) has one vote. Interested parties can be added at any time, but must be added at least one week before any election to have a vote. Any contributor from an organization may cast the vote for that organization. Each organization can cast one vote for a co-chair candidate in each of the communities (Kubernetes (K8s) Community, Service Provider (SP) Community, and CNF Developer (Dev) Community), three total votes.
```
Comment (xmulligan) [inline GOVERNANCE.md:60]:
```suggestion

### Voting Procedure
```

**Comment by xmulligan** [comment]
@taylor those weren't supposed to be deleted, but I made this PR before #146 was merged into main so it looks like I deleted them. I've added them back in. Otherwise I think it is ready to merge and it has 5 approvals

**Review by Smitholi67 — APPROVED** [review]

**Pull request merged**